conventions/documentation/badges.md

```diff
@@ -1,26 +1,55 @@
 ---
-parent: Conventions
-version: 4.2
+name: Badges
+menu: Indicators
 ---
 
 # Badges
 
+There are a variety of badges that are leveraged across the OpenShift platform.
+
+
+
 ## Resource badges
 
 + Colored badges are added to the left of all Kubernetes resources (including Custom Resources) in the console
 + Badges help users quickly identify resource type with a 1-3 letter abbreviation
-+ There are custom resource definitions that exceed 3 letter abbreviations but we try to avoid this for general pre-defined resources 
++ There are custom resource definitions that exceed 3 letter abbreviations but we try to avoid this for general pre-defined resources
```

Comment:
@andybraren do we want to note anything here regarding Operators & CRs?

Reply:
@andybraren fyi making you aware of Serena's comment above. Is there anything simple we could add at this time?

Reply:
Not that I can think of. I know OLM operators add CRs, but do their badge conventions differ in any way? Their color is always blue (I think) but that's covered further below.